src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -120,10 +139,10 @@ public class NodeSettings
         public DataFolder DataFolder { get; set; }
 
         /// <summary>Path to the data directory.</summary>
-        public string DataDir { get; set; }
+        public string DataDir { get; private set; }
```

Comment:
Can you just say in the documentation here and in ConfigurationFile that's it's read-only and that it needs to be set in the args?

Reply:
Done.